opendbc/safety/modes/rivian.h

```diff
@@ -94,10 +85,8 @@ static void rivian_rx_hook(const CANPacket_t *msg) {
 
   if (msg->bus == 2U) {
     // Cruise state
-    if (msg->addr == 0x100U) {
```

Comment:
don't like this one. it really hurts locality and readability. let's rethink how to approach this